persistence/nhibernate/scripting.md

```diff
@@ -26,3 +26,29 @@ Where:
 
  * `<subscriberAddress>` is the address of the subscriber. E.g. `My.Endpoint@subscriber-machine`.
  * `<subscriptionTable>` is the configured subscription table for NHibernate Persistence. By default this is `dbo.Subscription`
+
+## Fix up TimeoutEntity_EndpointIdx index
+
+Previous versions of NServiceBus had an issue which caused the `TimeoutEntity_EndpointIdx` index to be either missing or created incorrectly. The new versions have built-in check and log warnings in cases when
+ * the `TimeoutEntity_EndpointIdx` index has incorrect column order
+ * the system could not find `TimeoutEntity_EndpointIdx` index
+
+In order to mitigate the problem use following scripts to drop the index (if created incorrectly)
```

Comment:
i suggest adding a colon at the end, including "the" and changing the order of the sentence a bit... : 

>In cases where the index was created incorrectly, use **the** following scripts to drop the index: